smithy-aws-protocol-tests/model/ec2Query/xml-lists.smithy

```diff
@@ -23,7 +23,8 @@ use smithy.test#httpResponseTests
 /// 4. XML lists with @xmlName on its members
 /// 5. Flattened XML lists.
 /// 6. Flattened XML lists with @xmlName.
-/// 7. Lists of structures.
+/// 7. Flattened XML lists with @xmlNamespace.
```

Comment:
`ec2Query` got the updates for lists, but not the new test for maps.

Reply:
EC2 does not utilize any maps, so there aren't any tests for it in EC2.